sounddevice.py

```diff
@@ -2349,6 +2376,101 @@ def __init__(self, channel_selectors):
             channelSelectors=self._selectors))
 
 
+class CoreAudioSettings(object):
+
+    def __init__(self, channel_map=None, change_device_parameters=False,
+                 fail_if_conversion_required=False,
+                 conversion_quality='max'):
+        """Mac Core Audio-specific input/output settings.
+
+        Objects of this class can be used as *extra_settings* argument
+        to `Stream()` (and variants) or as `default.extra_settings`.
+
+        Parameters
+        ----------
+        channel_map : sequence of int, optional
+            Support for opening only specific channels of a Core Audio
+            device.  *channel_map* is a sequence of integers specifying
+            the (zero-based) channel numbers to use.  Note that
+            *channel_map* is treated differently between input and
+            output channels.
+
+            For input devices, *channel_map* is a list of integers
+            specifying the (zero-based) channel numbers to use.
+
+            For output devices, *channel_map* must have the same length
+            as the number of output channels of the device.  Specify
+            unused channels with -1, and a 0-based index for any desired
+            channels.
+
+            See the example below.  For additional information, see
+            https://app.assembla.com/spaces/portaudio/git/source/master/src/hostapi/coreaudio/notes.txt
+        change_device_parameters : bool, optional
+            If ``True``, allows PortAudio to change things like the
+            device's frame size, which allows for much lower latency,
+            but might disrupt the device if other programs are using it,
+            even when you are just Querying the device.  ``False`` is
+            the default.
+        fail_if_conversion_required : bool, optional
+            In combination with the above flag, ``True`` causes the
+            stream opening to fail, unless the exact sample rates are
+            supported by the device.
+        conversion_quality : {'min', 'low', 'medium', 'high', 'max'}, optional
+            This sets Core Audio's sample rate conversion quality.
+            ``'max'`` is the default.
+
+        Example
+        -------
+        This example assumes a device having 6 input and 6 output
+        channels.  Input is from the second and fourth channels, and
+        output will be sent to the third and fifth:
```

Comment:
This may be clear already, but to make it even more obvious, you could write something like "... and the two output channels will be sent to the third and fifth output channel of the device, respectively".

Or would this be more confusing?